lib/fastlane/plugin/ddg_apple_automation/helper/ddg_apple_automation_helper.rb

```diff
@@ -392,6 +392,11 @@ def self.compute_tag(is_prerelease)
           promoted_tag = "#{version}-#{build_number}"
         end
 
+        if platform_suffix && !platform_suffix.empty?
+          tag = "#{tag}+#{platform_suffix}"
+          promoted_tag = "#{promoted_tag}+#{platform_suffix}" unless is_prerelease
+        end
```

Comment:
We'll later update this to not be optional, requiring that every tag is computed with a platform.